assets/js/modules/analytics/components/settings/SettingsEdit.test.js

```diff
@@ -91,6 +91,12 @@ describe( 'SettingsEdit', () => {
 		const { accountID, propertyID } = existingTag;
 
 		registry.dispatch( CORE_MODULES ).receiveGetModules( [] );
+		registry
+			.dispatch( CORE_MODULES )
+			.receiveCheckModuleAccess(
+				{ access: true },
+				{ slug: 'analytics' }
+			);
```

Comment:
This shouldn't be needed I don't think as the test isn't about setting up with different users. I wonder if we need to update the modules fixture used by `provideModules` to include the `owner.id: 1` for all the connected modules instead. Either way, we should only need to ensure the owner ID or current user is set here so that access checking isn't needed/relevant.

Reply:
Forgot to remove this check. In any case, the existing tests now pass without any changes once again due [to us checking if `hasModuleAccess === false` instead of  `! hasModuleAccess` ](https://github.com/google/site-kit-wp/pull/5301#discussion_r887790859).